src/adapter/src/catalog/builtin.rs

```diff
@@ -1788,6 +1788,17 @@ pub static MZ_CLUSTER_REPLICA_FRONTIERS: Lazy<BuiltinTable> = Lazy::new(|| Built
     is_retained_metrics_relation: false,
 });
 
+pub static MZ_SUBSCRIBES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_subscribes",
+    schema: MZ_INTERNAL_SCHEMA,
+    desc: RelationDesc::empty()
+        .with_column("id", ScalarType::String.nullable(false))
+        .with_column("user", ScalarType::String.nullable(false))
+        .with_column("cluster_id", ScalarType::String.nullable(false))
+        .with_column("started_at", ScalarType::TimestampTz.nullable(false)),
```

Comment:
Not a strong opinion, since there seems to be no standard in the system catalog, but `created_at` could be more generic and future-standard-proof than `started_at`.

Reply:
Before we do another roundtrip: What should the `depends_on` column be named?

Reply:
We could follow the lead of `mz_object_dependencies` and use `referenced_object_id`? Or `ids`, since it can depend on multiple objects.